variables.tf

```diff
@@ -77,4 +77,16 @@ variable "force_delete" {
   type        = bool
   description = "Whether to delete the repository even if it contains images"
   default     = false
-}
\ No newline at end of file
+}
+
+variable "enable_registry_replication" {
+  type        = bool
+  description = "Set to 'true' to enable registry replication"
+  default     = false
+}
+
+variable "replication_configuration_rules" {
+  description = "The replication rules for a replication configuration. A maximum of 10 are allowed per"
+  type        = any
```

Comment:
@dmitrijn can you please provide this as a `list(object{ ... })` and bump this to require terraform 1.3+. We'd like to start doing that I believe and this is one place where I think that makes sense considering the variable's complexity.

Reply:
fixed